### PR TITLE
Fix expression and processes table focus

### LINKED ISSUE #454: focus in process list table
- apply new focus styles to the table
- make sure edit button focuses correctly when using TAB

### LINKED ISSUE #543: no focus in expression input fields
There is no blue focus border in expression input fields.

### PR DESCRIPTION
**Pull request by PPiedel**
This resolves #543 
This resolves #454

### CODE REVIEW

File: ui/client/components/table/TableRowIcon.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+  const {glyph, title, onClick, onKeyPress} = props
+
+  return <Glyphicon glyph={glyph}
+                    title={title}
+                    onClick={onClick}
+                    className={"processes-table-row-icon"}
+                    tabIndex={0}
+                    onKeyPress={event => {
+                      if (event.key === "Enter") {
```
Comment (lciolecki) [inline ui/client/components/table/TableRowIcon.js]: "Enter" should be defined as const :)

Comment (PPiedel) [reply ui/client/components/table/TableRowIcon.js]: Done

File: ui/client/components/graph/node-modal/editors/expression/ExpressionSuggest.js [hunk 2]
```diff
@@ -152,7 +154,9 @@ class ExpressionSuggest extends React.Component {
                          fontSize: 16,
                          fontFamily: "'Monaco', 'Menlo', 'Ubuntu Mono', 'Consolas', 'source-code-pro', monospace", //monospace font seems to be mandatory to make ace cursor work well,
                          readOnly: inputProps.readOnly
-                       }}/>
+                       }}
+                       onFocus={(_) => this.setEditorFocus(true)}
```
Comment (lciolecki) [inline ui/client/components/graph/node-modal/editors/expression/ExpressionSuggest.js]: There should be `onFocus={this.setEditorFocus(true)}` and definition should look: `setEditorFocus = (focus) => () => {}`. Please do in analogic way with onBlur.

File: ui/client/components/table/TableRowIcon.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  const {glyph, title, onClick, onKeyPress} = props
+
+  const enterKey = "Enter"
+
+  return <Glyphicon glyph={glyph}
+                    title={title}
+                    onClick={onClick}
+                    className={"processes-table-row-icon"}
+                    tabIndex={0}
+                    onKeyPress={event => {
```
Comment (lciolecki) [inline ui/client/components/table/TableRowIcon.js]: Please create arrow function: 

`
onKeyPress = (event) => {
 if (event.key === enterKey) {
   onClick()
 }
}
`

File: ui/client/containers/Archive.js [hunk 4]
```diff
@@ -98,10 +98,10 @@ class Archive extends BaseProcesses {
                     <Date date={process.modificationDate}/>
                   </Td>
                   <Td column="view" className="edit-column">
-                    <Glyphicon
+                    <TableRowIcon
                       glyph="eye-open"
                       title={"Show " + (process.isSubprocess ? "subprocess" : "process")}
-                      onClick={this.showProcess.bind(this, process)}
+                      onClick={(_) => this.showProcess(process)}
```
Comment (lciolecki) [inline ui/client/containers/Archive.js]: Please create arrow function ;)

File: ui/client/containers/Processes.js [hunk 5]
```diff
@@ -159,20 +158,20 @@ class Processes extends BaseProcesses {
...
                 </Td>
                 <Td column="edit" className="edit-column">
-                  <Glyphicon glyph="edit"
-                             title="Edit process"
-                             onClick={this.showProcess.bind(this, process)}
-                             className={"processes-table-row-icon"}
+                  <TableRowIcon
+                    glyph="edit"
+                    title="Edit process"
+                    onClick={(_) => this.showProcess(process)}
```
Comment (lciolecki) [inline ui/client/containers/Processes.js]: Arrow function cdn..

File: ui/client/components/graph/node-modal/editors/expression/ExpressionSuggest.js (hunk 2 above)
Comment (PPiedel) [reply ui/client/components/graph/node-modal/editors/expression/ExpressionSuggest.js]: Done

File: ui/client/components/table/TableRowIcon.js (hunk 3 above)
Comment (PPiedel) [reply ui/client/components/table/TableRowIcon.js]: Done

File: ui/client/containers/Archive.js (hunk 4 above)
Comment (PPiedel) [reply ui/client/containers/Archive.js]: Done

File: ui/client/containers/Processes.js (hunk 5 above)
Comment (PPiedel) [reply ui/client/containers/Processes.js]: Done

**Pull request merged**